docs/data-sources/virtualmachine.md

```diff
@@ -54,10 +56,14 @@ data "harvester_virtualmachine" "opensuse154" {
 - `restart_after_update` (Boolean) restart vm after the vm is updated
 - `run_strategy` (String) more info: https://kubevirt.io/user-guide/virtual_machines/run_strategies/
 - `secure_boot` (Boolean) EFI must be enabled to use this feature
-- `ssh_keys` (List of String)
+- `ssh_keys` (List of String) The `ssh_keys` are added to `cloudinit.user_data` if:
+1. Both `cloudinit.user_data_base64` and `cloudinit.user_data_secret_name` are empty.
+2. There is no `ssh_authorized_keys` field in `cloudinit.user_data`.
 - `start` (Boolean, Deprecated)
 - `state` (String)
-- `tags` (Map of String)
+- `tags` (Map of String) The `ssh-user` is added to `cloudinit.user_data` if:
```

Comment:
I’m wondering why we chose to introduce `ssh-user` as a tag instead of adding a new field. One possible reason could be a rule that new fields are not added in minor releases, is that the case?

Reply:
I just realized this PR is a bugfix, the `ssh-user` was defined in ancient times.